setup.py

```diff
@@ -52,7 +52,8 @@
         'simplejson',
         'twitter',
         'plone.app.mosaic',
-        'pyquery'
+        'pyquery',
+        'plone.app.imagecropping'
```

Comment:
```suggestion
        'plone.app.imagecropping',
```
in future, avoid two line changes in such a case. (the whole code could get a black/isort overhaul..., but that's OT here.)